dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -74,6 +74,58 @@ class CoursesControllerTest < ActionController::TestCase
     assert_redirected_to '/courses/csd-2018'
   end
 
+  test "show: redirect to latest stable version in course family for logged out user" do
+    sign_out @teacher
+    create :course, name: 'csp-2017', family_name: 'csp', version_year: '2017'
+    create :course, name: 'csp-2018', family_name: 'csp', version_year: '2018'
```

Comment:
a comment indicating that all `course` versions (as opposed to `script` versions) are currently stable would be helpful here.